ios/ItaliaApp.xcodeproj/project.pbxproj

```diff
@@ -1413,10 +1415,11 @@
 					"-ObjC",
 					"-lc++",
 				);
-				PRODUCT_BUNDLE_IDENTIFIER = it.teamdigitale.app.Italia;
+				PRODUCT_BUNDLE_IDENTIFIER = com.ibm.app.Italia;
```

Comment:
Whoops, needed to change domain to test things on an actual device -- will revert this to its original value